### ISSUE TITLE
index.js

### ISSUE THREAD

**Issue by lilpakky** [Fweak/CodeClaimer#4]
hi, 
when i try to open the index.js file, it pops up with an error 
![image](https://user-images.githubusercontent.com/76228074/102658008-a5467c00-416e-11eb-8c54-41fafb43d777.png)
Could you tell me how to fix this please , thanks

**Comment by Fweak** [comment]
You are required  to use [NodeJS](https://nodejs.org).
It's a JavaScript Runtime thingy

**Comment by lilpakky** [comment]
Do you have discord or something so you can tell me how to do it

**Comment by lilpakky** [comment]
I have Node Js installed but don’t know how to use it

**Comment by Fweak** [comment]
Try running: `npm start` when u first open cmd in the directory!

**Comment by lilpakky** [comment]
![image](https://user-images.githubusercontent.com/76228074/102692975-1f2e4200-420f-11eb-8074-01736aa3c542.png)

**Comment by lilpakky** [comment]
nevermind i resolved it and downloaded axios

**Comment by lilpakky** [comment]
how do i maximise effiiency? do i just join lots of servers ?

**Comment by Fweak** [comment]
Yeah, I think it has a j4j option in the config.json (Don't really remember). Toggling it will let ur "seakers" (the alt tokens)  try and join any invite sent in the chats they are in.

**Comment by lilpakky** [comment]
j4j isnt working apprently

**Comment by lilpakky** [comment]
![image](https://user-images.githubusercontent.com/76228074/102713472-688a9a00-42c0-11eb-846d-4ad143172dae.png)

**Comment by Fweak** [comment]
Sorry seems my retarded ass sucks at doing simple shit. So I updated the code. 
Just re-download the project and  you should be good  to go!

**Comment by lilpakky** [comment]
doesnt work yet
![image](https://user-images.githubusercontent.com/76228074/102765010-f2476f80-4373-11eb-8d33-355cd5989dd9.png)

**Comment by Fweak** [comment]
Make sure you are CD'd in the correct path to the project

**Comment by lilpakky** [comment]
![image](https://user-images.githubusercontent.com/76228074/102883017-36f00b00-4447-11eb-9da8-755902da2fde.png)
different error now

**Comment by Fweak** [comment]
Update the index.js file. You can just copy and paste it to urs

**Comment by lilpakky** [comment]
![image](https://user-images.githubusercontent.com/76228074/102981558-0a002e80-4501-11eb-889c-9031d332a8b9.png)
 this is the new problem

**Comment by lilpakky** [comment]
it says all the invite links are invalid

**Comment by therealtear** [comment]
> it says all the invite links are invalid

nigga ur autistic
